lte/gateway/configs/spgw.yml

```diff
@@ -14,6 +14,7 @@
 s11_iface_name: "lo"  # 'sgw_s11_ip' will be filled automatically
 s1u_iface_name: "eth1"  # 's1_ip' will be filled automatically
 sgw_s5s8_up_iface_name: "eth0"  # 'sgw_s5s8_up_ip' will be filled automatically
+sgw_s5s8_up_non_nat_iface_name: "uplink_br0"  # 'sgw_s5s8_up_ip' will be filled automatically
```

Comment:
```
sgw_s5s8_up_iface_name
sgw_s5s8_up_iface_name_non_nat
```

What about using same name but with a suffix to highlight the idea they are the same interface (s5s8) but on different configurations